### PR TITLE
Fix geocoding batched flag

### PR DESCRIPTION
**Pull request by rafatower**
@Kartones can you please review? (check the commit messages and ask me if any doubt)

Thanks!

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4002/))

**Comment by rafatower** [comment]
(I broke the tests with a silly thing related to config, not code, gonna fix it in a sec)

File: app/models/geocoding.rb [hunk 1]
```diff
@@ -112,7 +112,7 @@ def run!
   end
 
   def run_geocoding!(processable_rows, rows_geocoded_before = 0)
-    self.update state: 'started', processable_rows: processable_rows, batched: table_geocoder.use_batch_process?
+    self.update state: 'started', processable_rows: processable_rows
```
Comment (Kartones) [inline app/models/geocoding.rb]: so we're not storing anymore if was batched? just asking to get context

**Comment by Kartones** [comment]
:+1:

File: app/models/geocoding.rb (hunk 1 above)
Comment (rafatower) [inline app/models/geocoding.rb]: hum, isn't it saved just by setting the attribute? `@batched = value` ?

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4003/))

File: app/models/geocoding.rb (hunk 1 above)
Comment (Kartones) [inline app/models/geocoding.rb]: ah yes, if you do a save in other places yes, but that `.update` IIRC only changes the values specified as parameters, is not the same as a `.save`. Again IIRC, I'm not 100% sure

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4004/))

**Pull request merged**